docs/thirdparty.rst

```diff
@@ -37,7 +37,20 @@ Third Party Applications
 .. content list start
 
 
-.. replace this line with the first entry
+LiveStreamLinkGUI
+---------------------
+
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
+    :alt: LiveStreamLinkGUI
+
+:Description: A lightweight BASH script
+:Type: Graphical User Interface
+:OS: |Linux| |MacOS| And any other OS that supports BASH, Zenity, and typical \*nix commands
```

Comment:
The [field list](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#field-lists) doesn't have to be strict here.
I would like it better if you would add a new line called "Notes" with the Zenity, BASH, etc dependency description there. This makes it look much cleaner. Thanks!